EIPS/eip-2876.md

```diff
@@ -0,0 +1,188 @@
+---
+eip: 2876
+title: Deposit contract and address standard
+author: Jonathan Underwood (@junderw)
+discussions-to: https://github.com/junderw/deposit-contract-poc/issues/1
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-08-13
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+This ERC defines a simple contract interface for managing deposits. It also defines a new address format that encodes the extra data passed into the interface's main deposit function.
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+Centralized exchanges and merchants (Below: "apps") require an address format for accepting deposits. Currently the address format used refers to an account (external or contract), but this creates a problem. It requires that apps create a new account for every invoice / user. If the account is external, that means the app must have the deposit addresses be hot wallets, or have increased workload for cold wallet operators (as each deposit account will create 1 value tx to sweep). If the account is contract, generating an account costs at least 60k gas for a simple proxy, which is cost-prohibitive.
+
+This ERC seeks to create a standard for deposit systems (and how wallets interface with those systems) that will alleviate these problems.
```

Comment:
The first paragraph should probably be removed or moved to motivation (see previous comment) which leaves only this sentence as the abstract.  Consider expanding it a bit to try to give more detail on how you achieve that goal, though without going all the way into the specification. I like to use the abstract as a place to provide a human readable description of the specification for people who are trying to evaluate if this standard is what they want, and then the specification section is highly technical and targeted at people who are trying to implement this specification.